style.css

```diff
@@ -0,0 +1,241 @@
+/* QUESTIONS TO ASK:  */
+/* 1. why is my heading section tagline and call to action not aligned?*/
+
+/* overall style */
+div {
+    /* background-color: navy; */
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+ul {
+    padding: 0px;
+}
+
+/* nativation bar style */
+nav {
+    background-color: pink;
+    height: 3em;
+    font-weight: bolder;
+    padding-top: 10px;
+}
+nav ul {
+    /* border: solid; */
+    list-style: none;
+    display: grid;
+    grid-template: 25% 75% / 70% 15% 15%;
+    /* grid-template-columns: 70% 15% 15%; */
+    /* grid-template-rows: 25% 75%; */
+}
+nav ul li:first-child{
+    /* border: solid; */
+    grid-column-start: 1;
+    grid-row-start: 2;
+}
+nav ul li:nth-of-type(2) {
+    /* border: solid; */
+    grid-column-start: 2;
+    grid-row-start: 2;
+}
+nav ul li:nth-of-type(3) {
+    /* border: solid; */
+    grid-column-start: 3;
+    grid-row-start: 2;
+}
```

Comment:
Great job! I like how you used grid here and nth-of-type selector. I noticed that you had 2 rows and only second one was used. Was it intentional?